styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -338,6 +323,12 @@ private void run() {
     } catch (Exception e) {
       cliOutput.printError(getStackTraceAsString(e));
       throw CliExitException.of(ExitStatus.UnknownError);
+    } finally {
+      try {
+        styxClient.close();
+      } catch (IOException ignored) {
```

Comment:
Override the `close()` method in `StyxClient` to not throw an exception instead.